packages/core/framework/src/build-tools/compiler.ts

```diff
@@ -440,6 +463,7 @@ export class Compiler {
    * a file has changed.
    */
   async developPluginBackend(onFileChange?: () => void) {
+    await this.#createPluginOptionsFile()
```

Comment:
@thetutlage just making sure: Can we guarantee that this file will never end up in the final build of the plugin which is published to NPM, or would it be safer to clean this up when the dev process is killed? If it ends up in the final NPM package, admin extensions would fail to load for the plugin.

Reply:
I see what you mean. Ideally, I expect the users to run `npm run build` at the time of publishing the plugin to npm. And that command will perform a cleanup + build and this file will never part of the published file.